pipeline/id_minter/docker-compose.yml

```diff
@@ -1,13 +1,12 @@
-version: '3.3'
 services:
   mysql:
-    image: "public.ecr.aws/docker/library/mysql:8.0"
+    image: "mysql:8.0"
     ports:
       - "3307:3306"
```

Comment:
The local.docker-compose.yml is update to use port mapping 3306:3306. Why the inconsistency?